cli/commands/plan.go

```diff
@@ -55,7 +55,9 @@ func (cmd *planHandler) getPlanName() string {
 	if len(cmd.PlanName) > 0 {
 		return cmd.PlanName
 	}
-	return "deploy"
+	// there is no (and should not be) a case where the plan name is requested here where it is not needed.
+	client.PrintMessageAndExit("Must specify a plan name")
```

Comment:
Maybe give a hint that 'deploy' is probably what they want.

`Please specify a plan name, e.g. 'deploy'`